.github/workflows/ci-goreleaser-ebpf-profiler.yaml

```
@@ -0,0 +1,37 @@
+name: CI - eBPF Profiler - GoReleaser
+
+on:
+  merge_group:
+  push:
+    branches: [main]
+    paths:
+      - "distributions/otelcol-ebpf-profiler/**"
+      - "cmd/**"
+      - ".github/**"
+      - "scripts/**"
+      - "Makefile"
+      - "go.mod"
+      - "go.sum"
+  pull_request:
+    branches: [main]
+    paths:
+      - "distributions/otelcol-ebpf-profiler/**"
+      - "cmd/**"
+      - ".github/**"
+      - "scripts/**"
+      - "Makefile"
+      - "go.mod"
+      - "go.sum"
+
+jobs:
+  check-goreleaser:
+    name: CI - eBPF Profiler - GoReleaser
+    uses: ./.github/workflows/base-ci-goreleaser.yaml
+    with:
+      distribution: otelcol-ebpf-profiler
+      config_file: ebpf-profiler-config
+      docker_run_options: '--privileged -v /sys/kernel/debug/:/sys/kernel/debug/'
```

The eBPF profiler needs to get access to the host pid and it does not require write access mapping to /sys/kernel/debug
```suggestion
      docker_run_options: '--privileged --pid=host -v /sys/kernel/debug/:/sys/kernel/debug/:ro'
```